receptorctl/receptorctl/cli.py

```diff
@@ -28,6 +27,17 @@ def parse_args(self, ctx, args):
                 param.required = False
             return super(IgnoreRequiredWithHelp, self).parse_args(ctx, args)
 
+def pprint(json_data):
+    click.echo(json.dumps(json_data, indent=4, sort_keys=True))
+
+def printWarning(message):
```

Comment:
Python function names should be `snake_case`. https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Fixed.